### DISCUSSION TITLE
Help with turns when button pressed in Xbox Controller.

### DISCUSSION THREAD

**Discussion by rTHoiMas** [pybricks/support#1697] [Q&A]
I want to make the robot turn when I press a button, but without pauses. Could you please let me know if this is possible? The below code works, but it executes in breaks and is not a smooth motion.

  
    if jleft[0]<0:
        drive_base.turn(10)
    elif jleft[0]>0:
        drive_base.turn(-10)

**Comment by laurensvalk** [comment]
You can use the `drive` method to start driving indefinitely, until you choose a different speed or stop it yourself.

**Reply by rTHoiMas** [reply]
That only drives forward. I can't turn with that

**Reply by laurensvalk** [reply]
The first argument is for forward speed. The second argument is for steering.

**Comment by BertLindeman** [comment]
Hi,
(if I may chime-in, Laurens)

Might this [example](https://pybricks.com/projects/tutorials/wireless/remote-control/drive-base/) give you a hint on how it can be done? 
It is not the xbox controller, but that part you probably already have under control.

Bert

**Reply by rTHoiMas** [reply]
Thank You! I am going out of town for the rest of the week, will be back on Sunday.
